clarity/src/vm/representations.rs

```diff
@@ -51,6 +51,8 @@ lazy_static! {
         "({})|({})",
         *STANDARD_PRINCIPAL_REGEX_STRING, *CONTRACT_PRINCIPAL_REGEX_STRING
     );
+    static ref CLARITY_NAME_NO_BOUNDARIES_REGEX_STRING: String =
```

Comment:
This needs boundaries.  Unbounded regexes whose evaluation is accessible via the network are a DDoS bug waiting to happen.

Reply:
All of the regex string I added end up being formatted together and exclusively used with boundaries ([here](https://github.com/stacks-network/stacks-core/blob/26c448727def84bbef34ba4d14ec4fd5244a2390/stackslib/src/net/api/getclaritymarfvalue.rs#L63-L66) and [here](https://github.com/stacks-network/stacks-core/blob/26c448727def84bbef34ba4d14ec4fd5244a2390/stackslib/src/net/api/getclaritymetadata.rs#L63-L68))

But since they are only used there, maybe I should build the regex more exclusively in these two files and not exposed in representation.rs. Would that be a better approach?